### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by leongdl135**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+ProfNUS is a desktop application which helps SOC Professors who have many modules with many students/TAs to manage. It is optimized for users who prefer CLI over GUI so that frequent tasks can be done faster by typing in commands.
+
+To find out more about ProfNUS, please visit out **[Product Website](https://ay2223s1-cs2103t-w11-2.github.io/tp/)**
```
Comment (bryanngzh) [inline README.md:9]: Should change "out" to "our".

**Review by bryanngzh — CHANGES_REQUESTED** [review]
Do update the typo on line 9 before merging. Thanks!

File: README.md (hunk 1 above)
Comment (leongdl135) [reply README.md]: Thanks updated

**Review by bryanngzh — APPROVED** [review]
LGTM!

**Pull request merged**